providers/org/mpra/apps.py

```diff
@@ -0,0 +1,10 @@
+from share.provider import OAIProviderAppConfig
+
+
+class AppConfig(OAIProviderAppConfig):
+    name = 'providers.org.mpra'
+    version = '0.0.1'
+    title = 'mpra'
+    long_title = 'Munich Personal RePEc Archive'
+    home_page = 'http://mpra.ub.uni-muenchen.de'
+    url = 'http://mpra.ub.uni-muenchen.de/perl/oai2'
```

Comment:
@erinspace does this URL require additional url params, such as metadata namespace?